### PR TITLE
Adding basic auth code for acceptance tests.

### PR DESCRIPTION
**Pull request by awais786**
ECOM-3850
On Stage there is basic authentication is on. Adding its authentication.

### CODE REVIEW

**Comment by awais786** [comment]
@rlucioni Please review this PR.

File: acceptance_tests/config.py [hunk 1]
```diff
@@ -48,6 +48,11 @@ def str2bool(s):
...
 LMS_PASSWORD = os.environ.get('LMS_PASSWORD')
 
+BASIC_AUTH_USERNAME = os.environ.get('BASIC_AUTH_USERNAME')
+BASIC_AUTH_PASSWORD = os.environ.get('BASIC_AUTH_PASSWORD')
+
 if ENABLE_OAUTH2_TESTS and not (LMS_ROOT_URL and LMS_USERNAME and LMS_PASSWORD):
     raise RuntimeError('Configuring LMS settings is required to run OAuth2 tests.')
 # END LMS CONFIGURATION
+
+#CREDENTIALS_ROOT_URL="https://stage-edx-credentials.edx.org" LMS_ROOT_URL="https://courses.stage.edx.org" LMS_USERNAME="awais" LMS_EMAIL="awais@edx.org" LMS_PASSWORD="1234567a" ACCESS_TOKEN="51fa9297a6a40bb7d492ab14549a5e3bf674e48a" USER_JWT_AUDIENCE="0a71ed8e1bde75af80e8" JWT_SECRET_KEY="c3bbeb72268042785a66fb5ed80344f532134664" ENABLE_OAUTH2_TESTS=True  PROGRAM_ID=12 make accept
```
Comment (rlucioni) [inline acceptance_tests/config.py]: Please remove this.

**Comment by rlucioni** [comment]
@awais786 I've updated this branch to removing your stage credentials from the diff. Please don't commit credentials.

:+1: otherwise.

**Comment by zubair-arbi** [comment]
Changes looks good. :+1:

**Pull request merged**